README.md

```diff
@@ -1,8 +1,47 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Vandana Damodara . Jarvis Consulting
 
+I am a qualified and experienced software engineer with three years of experience in the industry. I had completed my  Bachelor of Technology degree in Electronics and Telecommunications. As a software developer, I believe that being able to work with/understanding lower levels of abstraction behind a program is a prerequisite to writing highly optimized code. I enjoy creative problem solving and getting exposure to multiple projects. I love working in a collaborative work environment. I am a Highly proficient, diligent, and excellent problem solver. As a Junior Data Engineer at Jarvis, I was able to participate in various challenging projects using an Agile/Scrum framework that allowed me to become proficient in technologies like Java, Git, SQL, Bash, Maven, and Docker. My experiences have taught me how to think outside the box when working, and I always strive to find the most optimized solution to the problem. I strive to stay current with relevant techniques and technologies
```

Comment:
did some minor fix

I am a qualified and experienced software engineer with three years of experience in the industry. I had completed my Bachelor of Technology degree in Electronics and Telecommunications. As a software developer, I believe that working with/understanding lower levels of abstraction behind a program is a prerequisite to writing highly optimized code. I enjoy creative problem solving and getting exposure to multiple projects. I love working in a collaborative work environment. I am a Highly proficient, diligent, and excellent problem solver. As a Junior Data Engineer at Jarvis, I participated in various challenging projects using an Agile/Scrum framework that allowed me to become proficient in technologies like Java, Git, SQL, Bash, Maven, and Docker. My experiences have taught me how to think outside the box when working, and I always strive to find the most optimized solution to the problem. I strive to stay current with relevant techniques and technologies.